partitioner/src/test/java/io/confluent/connect/storage/partitioner/TimeBasedPartitionerTest.java

```diff
@@ -583,6 +580,37 @@ public void testRecordFieldTimeStringExtractor() {
     validateEncodedPartition(encodedPartition);
   }
 
+  @Test
+  public void testRecordFieldTimeStringExtractorRecord() {
```

Comment:
```suggestion
  public void testRecordFieldTimeAsMillisStringExtractorRecord() {
```

Reply:
would be nice to rename the existing StringExtractor tests to indicate it is for ISO dates as well